quickwit/quickwit-doc-mapper/src/default_doc_mapper/field_mapping_entry.rs

```diff
@@ -353,6 +353,13 @@ impl TextIndexingOptions {
         }
     }
 
+    fn from_parts_concatenate(
+        tokenizer: Option<QuickwitTextTokenizer>,
+        record: Option<IndexRecordOption>,
+    ) -> anyhow::Result<Self> {
+        Self::from_parts_text(true, tokenizer, record, false).map(Option::unwrap)
```

Comment:
Nitpick:
`.map(Option::unwrap)` is difficult to read.

```suggestion
        let text_index_options_opt: Option<TextIndexingOptions> =  TextIndexingOptions::from_parts_text(true, tokenizer, record, false)?;
        let text_index_options = text_index_options_opt.expect("an indexed field should never result in a `None` `TextIndexingOptions`.");
        Ok(text_index_options)
```